scripts/local_testnet/README.md

```diff
@@ -5,58 +5,105 @@ This setup can be useful for testing and development.
 
 ## Requirements
 
-The scripts require `lcli`, `lighthouse`, `geth`, `bootnode` to be installed on `PATH`.
+The scripts require `lcli`, `lighthouse`, `geth`, `bootnode` to be installed on `PATH` (run `echo $PATH` to view all `PATH` directories).
 
 
 MacOS users need to install GNU `sed` and GNU `grep`, and add them both to `PATH` as well.
 
-From the
-root of this repository, run:
+The first step is to install Rust and dependencies. Refer to the [Lighthouse Book](https://lighthouse-book.sigmaprime.io/installation-source.html#dependencies) for installation. We will also need [jq](https://jqlang.github.io/jq/), which can be installed with `sudo apt install jq`.
+
+Then, we clone the Lighthouse repository:
+```bash
+cd ~
+https://github.com/sigp/lighthouse.git
+cd lighthouse
+```
+We are now ready to build Lighthouse. Run the command:
 
 ```bash
 make
 make install-lcli
 ```
 
+This will build `lighthouse` and `lcli`. For `geth` and `bootnode`, go to [geth website](https://geth.ethereum.org/downloads) and download the `Geth & Tools`. For example, to download and extract `Geth & Tools 1.13.1`:
+
+```bash
+cd ~
+curl -LO https://gethstore.blob.core.windows.net/builds/geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
+tar xvf geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
+```
+
+After extraction, copy `geth` and `bootnode` to the `PATH`. A typical directory is `/usr/local/bin`.
+
+```bash
+cd geth-alltools-linux-amd64-1.13.1-3f40e65c
+sudo cp geth bootnode /usr/local/bin
+```
+
+After that We can remove the downloaded files:
+
+```bash
+cd ~
+rm -r geth-alltools-linux-amd64-1.13.1-3f40e65c geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
+```
+
+We are now ready to start a local testnet.
+
 ## Starting the testnet
 
-Modify `vars.env` as desired.
+To start a testnet using the predetermined settings:
+
+```bash
+cd ~
+cd /lighthouse/scripts/local_testnet
```

Comment:
I think the path should be:

```suggestion
cd ~
cd lighthouse/scripts/local_testnet
```

Although I think it's probably fine to just document commands that runs  "From the root of the repository" to keep things simple.

Reply:
I miss a `.`, fix in [00f68f2](https://github.com/sigp/lighthouse/pull/4749/commits/00f68f22b786e570c1e9b7ae8c92a8ace862ddec), hopefully that's good now